analyzer/pkg/rpm/rpm_test.go

```diff
@@ -598,3 +598,46 @@ func TestParseRpmInfo(t *testing.T) {
 		})
 	}
 }
+
+func Test_splitFileName(t *testing.T) {
+	type args struct {
+		filename string
+	}
+	tests := []struct {
+		name     string
+		args     args
```

Comment:
In this case, we can simply define `filename` here.

```suggestion
		filename string
```